interfaces/mount/mountinfo.go

```diff
@@ -53,53 +53,56 @@ func ParseInfoEntry(s string) (InfoEntry, error) {
 	// 36 35 98:0 /mnt1 /mnt2 rw,noatime master:1 - ext3 /dev/root rw,errors=continue
 	// (1)(2)(3)   (4)   (5)      (6)      (7)   (8) (9)   (10)         (11)
 	if len(fields) < 10 {
-		return e, fmt.Errorf("incorrect number of fields, expected at least 10 but found %d", len(fields))
+		return nil, fmt.Errorf("incorrect number of fields, expected at least 10 but found %d", len(fields))
```

Comment:
No, before those were not pointers so I returned an empty object. In this branch, because I added an array into the structure, I now return a pointer to entry.